include/aspect/postprocess/gravity_point_values.h

```diff
@@ -157,65 +157,70 @@ namespace aspect
         unsigned int quadrature_degree_increase;
 
         /**
-         * Parameter for the uniform distribution sampling scheme:
+         * Parameter for the fibonacci spiral sampling scheme:
+         */
+        unsigned int n_points_spiral;
+
+        /**
+         * Parameter for the map sampling scheme:
          * Gravity may be calculated for a sets of points along the radius (e.g. depth
          * profile) between a minimum and maximum radius. Number of points along the radius
          * is specified with n_points_radius.
          */
         unsigned int n_points_radius;
 
         /**
-         * Parameter for the uniform distribution sampling scheme:
+         * Parameter for the map and fibonacci spiral sampling scheme:
```

Comment:
where is this used in the fibonacci spiral? I think this should to to the `n_points_radius` variable.